package.json

```diff
@@ -2,13 +2,10 @@
   "name": "@sayari/trellis",
   "version": "0.7.0-rc.0",
   "description": "High-performance network visualization library, rendering to WebGL as well as other targets",
-  "source": "src/index.ts",
-  "main": "./dist/main.js",
-  "module": "./dist/module.js",
-  "types": "./dist/types.d.ts",
-  "files": [
-    "dist"
-  ],
+  "source": "src/index.umd.ts",
+  "main": "./legacy/main.js",
```

Comment:
rather than name the umd directory `legacy` let's just name it `umd` (umd is just a different module format, but isn't necessarily legacy or meant to work only w/ older browsers. e.g. we introduced it to work w/ the code sandbox we were using to host examples b/c that was easier than getting it to work w/ npm directly). and it looks like the umd build output is `index.js` not `main.js`, so just need to be `./dist/umd/index.js`

Reply:
also, not positive, but I think `types` needs to be `./dist/index.d.ts`.

Reply:
finally, rather than main/module, should we use get rid of `main` and `module` and just use `exports`? https://nodejs.org/api/packages.html#package-entry-points